include/boost/nowide/config.hpp

```diff
@@ -50,15 +50,15 @@
 
 //! @endcond
 
-#if(defined(__GNUC__) || defined(__MINGW32__))
+#if(defined(__GNUC__) || defined(__MINGW32__) && !defined(__APPLE__))
 #define BOOST_NOWIDE_FTELL64 ftello64
 #define BOOST_NOWIDE_FSEEK64 fseeko64
-#elif(defined(_MSC_VER) && defined(_MSC_VER) >= 1400)
-#define BOOST_NOWIDE_FTELL64 _ftelli64
-#define BOOST_NOWIDE_FSEEK64 _fseeki64
 #elif(defined(__APPLE__))
 #define BOOST_NOWIDE_FTELL64 ftello
 #define BOOST_NOWIDE_FSEEK64 fseeko
+#elif(defined(_MSC_VER) && defined(_MSC_VER) >= 1400)
```

Comment:
I guess you meant to write ` && _MSC_VER >= 1400`, but you can omit this completetly as older MSVCs are not C++11 conformant enough anyway